android/src/main/java/com/reactnativeescposprinter/EscPosPrinterDiscoveryModule.java

```diff
@@ -261,6 +273,14 @@ public void sendDiscoveredDataToJS() {
 
   @ReactMethod
   private void discover(final ReadableMap paramsMap, Promise promise) {
+    mFindFirst = false;
+
+    if (paramsMap != null) {
+      if (paramsMap.hasKey("findFirstAndroid")) {
```

Comment:
Can we extract please such params assignments ( findFirstAndroid, usbSerialNumber and etc. ) to a separate method called? smth like:

**defineSettingsFromParamsMap**

And then:
```
@ReactMethod
  private void discover(final ReadableMap paramsMap, Promise promise) {
    mFindFirst = false;

    defineSettingsFromParamsMap(paramsMap)


```

Reply:
will do.